src/XeroPHP/Enums/Accounting/Organisation/OrganisationType.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace XeroPHP\Enums\Accounting\Organisation;
+
+class OrganisationType
```

Comment:
https://developer.xero.com/documentation/api/accounting/types/#organisation

In the docs it has 

ACCOUNTING_PRACTICE
CHARITY
CLUB_OR_SOCIETY
COMPANY
INDIVIDUAL
LOOK_THROUGH_COMPANY
NOT_FOR_PROFIT
PARTNERSHIP
S_CORPORATION
SELF_MANAGED_SUPERANNUATION_FUND
SOLE_TRADER
SUPERANNUATION_FUND
TRUST